server/routes/serviceProviderReferrals/postSessionFeedbackCheckAnswersPresenter.ts

```diff
@@ -77,7 +78,16 @@ export default class PostSessionFeedbackCheckAnswersPresenter {
     }
   }
 
-  get sessionDetailsSummary(): SummaryListItem[] {
-    return this.attendancePresenter.sessionDetailsSummary
-  }
+  readonly sessionDetailsSummary: SummaryListItem[] = [
+    {
```

Comment:
We could still have an `attendancePresenter` property that we instantiate inside this class — to be used just for deriving the value of `sessionDetailsSummary` —  _in addition to_ `feedbackAnswersPresenter`. Then we could keep it DRY (if that's the goal).

Reply:
I've decided to keep it separate and less DRY for now, but explained why in the commit message. Once we know a bit more about the fields we're displaying across the SP and PP journeys, I'm wondering if we want to turn it into a presenter of its own.